include/evmmax/evmmax.hpp

```diff
@@ -8,6 +8,22 @@
 namespace evmmax
 {
 
+/// Compute the modulus inverse for Montgomery multiplication, i.e. N': mod⋅N' = 2⁶⁴-1.
+///
+/// @param mod0  The least significant word of the modulus.
+inline constexpr uint64_t compute_mod_inv(uint64_t mod0) noexcept
```

Comment:
```suggestion
constexpr uint64_t compute_mod_inv(uint64_t mod0) noexcept
```

I believe `constexpr` implies `inline`.